### PR TITLE
Update developer and user guide

### PR DESCRIPTION
**Pull request by TPH777**
#63 #49

### CODE REVIEW

**Comment by rharwo** [comment]
lgtm

File: docs/Diagrams/storageLoad.png
Comment (NigelYeoTW) [inline docs/Diagrams/storageLoad.png:1]: LoginStorage load method does not have any parmaeters Ui class is all static methods

File: docs/UserGuide.md [hunk 1]
```diff
@@ -317,16 +315,29 @@ The `edit` command allows you to edit the amount, description, or date of an exi
...
+	Bye. Hope to see you again soon!
+	____________________________________________________________
+```
 
 ## FAQ
 
 **Q**: How do I transfer my data to another computer? 
 
-**A**: {your answer here}
+**A**: Simply transfer `incomes.txt`, `spendings.txt` and `password.txt` files to the folder that the program is at.
```
Comment (NigelYeoTW) [inline docs/UserGuide.md:338]:
```suggestion
**A**: Simply transfer `incomes.txt`, `spendings.txt` and `password.txt` files to the directory containing the jar file.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -104,11 +103,10 @@ output
 #### Adding an income:
 
 Adds an entry into user income list. Entry will be displayed at the latest index. <br>
-Run the [`list incomes`](#listing-all-incomes) command to display the list with the new entry.
+Run the [`list incomes`](#listing-all-incomes) command to display the list with the new entry. <br>
 Date is optional and can be added to the end of the input, default would be set to current date. A date is enclosed in slashes. <br>
 Tag is optional and can be added to the end of the input. A Tag is enclosed in asterisks. <br>
```
Comment (NigelYeoTW) [inline docs/UserGuide.md:109]:
```suggestion
Recurrence Frequency is optional and can be added to the end of the input. A Recurrence Frequency is enclosed in tilde. <br>

```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -72,11 +72,10 @@ Note that this is also the startup page for returning users
 #### Adding a spending:
 
 Adds an entry into user spending list. Entry will be displayed at the latest index. <br>
-Run the [`list spendings`](#listing-all-spendings) command to display the list with the new entry.
+Run the [`list spendings`](#listing-all-spendings) command to display the list with the new entry. <br>
 Date is optional and can be added to the end of the input, default would be set to current date. A date is enclosed in slashes. <br>
 Tag is optional and can be added to the end of the input. A Tag is enclosed in asterisks. <br>
```
Comment (NigelYeoTW) [inline docs/UserGuide.md:78]:
```suggestion
Recurrence Frequency is optional and can be added to the end of the input. A Recurrence Frequency is enclosed in tilde. <br>
```

**Review by NigelYeoTW — COMMENTED** [review]
overall lgtm just some additions for recurrence in add commands and also minor error in sequence diagram.

**Pull request merged**